### ISSUE TITLE
CSInterface.openURLInDefaultBrowser cannot open a specified page in a PDF file

### ISSUE THREAD

**Issue by zhangusc** [Adobe-CEP/CEP-Resources#351]
I try using **CSInterface.openURLInDefaultBrowser** to open a local pdf file and go directly to a specified page inside that pdf file.
The method works for a PDF file on a web URL, but not for opening a PDF file on the disk (inside the extension folder)

CSInterface.openURLInDefaultBrowser("https://www.esri.com/content/dam/esrisites/en-us/media/brochures/arcgis-enterprise-functionality-matrix.pdf#page=5")
Open the PDF in the browser and display page 5

Download the pdf to your CEP extension folder on the disk and open the file 
CSInterface.openURLInDefaultBrowser("[file:///C:/Program%20Files%20(x86)/Common%20Files/Adobe/CEP/extensions/[YourExtensionFolder]/arcgis-enterprise-functionality-matrix.pdf#page=5](url)")
Open the PDF in the browser but display page 1. The hashtag in the URL got lost.

**Comment by zhangusc** [comment]
@ErinFinnegan Could you pass this to the CEP team and have someone investigate the issue?

**Comment by ErinFinnegan** [comment]
Yes, absolutely! The product teams have been swamped lately doing updates for M1/Apple Silicon, but there should be more bandwidth now. (The two monitor bug is slated for this month as well.)

**Comment by sujaisivanandan** [comment]
@zhangusc I dont think this was working earlier either.Was it? It seems to be more of an enhancement to me.

**Comment by zhangusc** [comment]
Thank you @sujaisivanandan for looking into it. I didn't check the issue on old releases. Enhancement/bug, either way, sounds good to me as long as it gets into your plan.
